xsd/siri_utility/siri_location-v2.0.xsd

```diff
@@ -211,13 +211,34 @@ Lower right corner.
   <xsd:sequence>
    <xsd:element name="Point" type="LocationStructure" minOccurs="2" maxOccurs="unbounded">
     <xsd:annotation>
-     <xsd:documentation>A geospatial point. +SIRI v2.0 
-
-.</xsd:documentation>
+     <xsd:documentation>A geospatial point. +SIRI v2.0</xsd:documentation>
     </xsd:annotation>
    </xsd:element>
   </xsd:sequence>
  </xsd:complexType>
+ <xsd:complexType name="FlexibleAreaStructure">
+  <xsd:annotation>
+   <xsd:documentation>Bounding box, circular area or gml:polyon of the area where stops of a flexible service are called. +SIRI v2.1
+A flexible area is used in cases where a pre-booked service allows pick-up/drop-off anywhere in a designated area and provides a possible interchange to a higher-frequency service.</xsd:documentation>
+  </xsd:annotation>
+  <xsd:choice>
+   <xsd:element name="BoundingBox" type="BoundingBoxStructure">
+    <xsd:annotation>
+     <xsd:documentation>Flexible area specified as a rectangular bounding box.</xsd:documentation>
+    </xsd:annotation>
+   </xsd:element>
+   <xsd:element name="CircularArea" type="LocationStructure">
```

Comment:
See comment on LocationStructure. 
I would prefer to put Radius as separate attribute inside CircularArea, instead of inside the LocationStructure.

Reply:
See Commit [abd9a81](https://github.com/SIRI-CEN/SIRI/pull/64/commits/abd9a81d05b94287158fe8d4e28029db2475e4de)